src/components/Card/Card.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {Link} from "react-router-dom"
+
+const CardButton = (props) => {
+  return 
+    <Link
+      to={props.link}
+      className={`card-btn link-btn ${
+        props.variant === "primary" ? "button-primary" : "button-secondary"
+      }`}
+    >
+      {props.info}
+    </Link>
+};
+
+const Card = (props) => {
+  return (
+      <div
+        className={`card-shadow ${
+          props.variant === "vertical" ? "card-vetical" : "card-horizontal"
+        } ${props.overlayText ? "text-overlay-container" : ''}
+        ${props.className? props.className : ""}`}
+      >
+        
+        <img
+          className="card-img"
+          src={props.img.src}
+          alt={props.img.alt}
+        />
+
+        {/* <CardImage src={props.img.src}
+          alt={props.img.alt}/> */}
```

Comment:
remove comment

Reply:
commented code is not working, it is there for future PRs